docs/content/Aggregations.md

```diff
@@ -46,12 +46,28 @@ Computes the sum of values as 64-bit floating point value. Similar to `longSum`
 
 #### `max` aggregator
 
-`max` computes the maximum metric value
+`max` computes the maximum metric value and returns a double.
 
 ```json
 { "type" : "max", "name" : <output_name>, "fieldName" : <metric_name> }
 ```
 
+#### `longMin` aggregator
+
+`longMin` computes the minimum among metric value from rows and Long.MAX_VALUE
```

Comment:
it might make sense for the documentation to be worded the same for both long and double?